ran.py

```diff
@@ -257,7 +257,6 @@ def run_ue(self):
                 '--band', '78',
                 '-C', f'{self.ssb_frequency}',
                 '--ue-txgain', '0',
-                '-A', f'{self.conf["timing_advance"]}',
```

Comment:
This needs to be conditional w.r.t to `self.arg.rfsim`, otherwise it will break on Colosseum/MCHEM